Sync `svg/path` tests from WPT upstream

Ahmad-S792 opened the pull request:
#### f22266b0d14d4a87b65e98ca4ba56c62d4b3ef83
<pre>
Sync `svg/path` tests from WPT upstream

<a href="https://bugs.webkit.org/show_bug.cgi?id=286638">https://bugs.webkit.org/show_bug.cgi?id=286638</a>
<a href="https://rdar.apple.com/143775466">rdar://143775466</a>

Reviewed by Antoine Quint.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/1b476ffbe5be1470c147442006c6002eea44e61f">https://github.com/web-platform-tests/wpt/commit/1b476ffbe5be1470c147442006c6002eea44e61f</a>

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/bearing/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/closepath/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/error-handling/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/interfaces/SVGAnimatedPathData-removed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/interfaces/SVGAnimatedPathData-removed.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/interfaces/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/property/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/bearing/absolute-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/bearing/relative-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/bearing/zero-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/closepath/segment-completing-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-circle-mutating-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-circle-mutating.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-mutating-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-negative-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-zero-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-percentage-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-rect-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-rect-mutating-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-percentage-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/error-handling/render-until-error-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/interfaces/SVGPathSegment-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/interfaces/SVGPathSegment.svg:
* LayoutTests/TestExpectations: Skip failing tests

&gt; Removed `pixel` dump test expectations (old):
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-mutating-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-negative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-zero-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/bearing/absolute-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/bearing/relative-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/bearing/zero-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/closepath/segment-completing-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/bearing/absolute-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/bearing/relative-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/bearing/zero-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/closepath/segment-completing-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/bearing/absolute-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/bearing/relative-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/bearing/zero-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/closepath/segment-completing-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-mutating-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-negative-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-zero-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-percentage-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/distance/pathlength-rect-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-percentage-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/bearing/absolute-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/bearing/relative-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/bearing/zero-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/closepath/segment-completing-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-mutating-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-negative-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-zero-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-percentage-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/distance/pathlength-rect-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/distance/pathlength-rect-mutating-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-percentage-expected.png: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/path/bearing/absolute-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/path/bearing/relative-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/path/bearing/zero-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/path/closepath/segment-completing-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-percentage-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-percentage-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/error-handling/render-until-error-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/property/priority-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-percentage-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-percentage-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/error-handling/render-until-error-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/property/priority-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/error-handling/render-until-error-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/error-handling/render-until-error-expected.png: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-percentage-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-percentage-expected.txt: Removed.

Canonical link: <a href="https://commits.webkit.org/289589@main">https://commits.webkit.org/289589@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/6a3cf535334806fb0bee4b08ace190fdb3b16b66

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/87442 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/6955 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/41814 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/92304 "Built successfully") | [❌ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/38181 "Failed to checkout and rebase branch from PR 39641") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/7337 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/15120 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/67541 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [❌ 🧪 win-tests](https://ews-build.webkit.org/#/builders/59/builds/38181 "Failed to checkout and rebase branch from PR 39641") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/90444 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/132/builds/5588 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/79119 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/47881 "Passed tests") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/133/builds/5377 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/33513 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/37297 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/75795 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/34376 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/94189 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/14607 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/122/builds/10714 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/76364 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/14861 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/74974 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/75578 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/18588 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/19963 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/18376 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/7545 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/14625 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/14368 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/17812 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/16151 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

graouts commented on LayoutTests/TestExpectations:
```diff
@@ -2187,6 +2187,15 @@ imported/w3c/web-platform-tests/svg/animations/repeat-event.svg [ Skip ]
+imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-negative.svg [ Skip ]
+imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-zero.svg [ Skip ]
+imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero.svg [ Skip ]
+imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-percentage.svg [ Skip ]
```
Why do we need to skip those?

Ahmad-S792 replied:
I can make them [ ImageonlyFailure ], I skipped them because we are failing them.

graouts replied:
I'll let you be the judge of what's best, but skipping them means that we wouldn't be able to determine potential progressions from the bots.

Ahmad-S792 replied:
True - I will update them post EWS to [ ImageonlyFailure ] run before merging.

Ahmad-S792 replied:
Changed to `ImageOnlyFailure`

Commit:
Sync `svg/path` tests from WPT upstream

https://bugs.webkit.org/show_bug.cgi?id=286638
rdar://143775466

Reviewed by Antoine Quint.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/1b476ffbe5be1470c147442006c6002eea44e61f

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/bearing/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/closepath/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/error-handling/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/interfaces/SVGAnimatedPathData-removed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/interfaces/SVGAnimatedPathData-removed.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/interfaces/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/property/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/bearing/absolute-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/bearing/relative-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/bearing/zero-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/closepath/segment-completing-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-circle-mutating-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-circle-mutating.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-mutating-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-negative-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-zero-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-percentage-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-rect-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-rect-mutating-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-percentage-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/error-handling/render-until-error-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/interfaces/SVGPathSegment-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/path/interfaces/SVGPathSegment.svg:
* LayoutTests/TestExpectations: Skip failing tests

> Removed `pixel` dump test expectations (old):
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-mutating-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-negative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-zero-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/bearing/absolute-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/bearing/relative-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/bearing/zero-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/closepath/segment-completing-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/bearing/absolute-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/bearing/relative-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/bearing/zero-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/closepath/segment-completing-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/bearing/absolute-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/bearing/relative-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/bearing/zero-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/closepath/segment-completing-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-mutating-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-negative-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-zero-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-percentage-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/distance/pathlength-rect-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-percentage-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/bearing/absolute-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/bearing/relative-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/bearing/zero-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/closepath/segment-completing-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-mutating-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-negative-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-zero-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-percentage-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/distance/pathlength-rect-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/distance/pathlength-rect-mutating-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-expected.png: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-percentage-expected.png: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/path/bearing/absolute-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/path/bearing/relative-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/path/bearing/zero-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/path/closepath/segment-completing-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-percentage-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-percentage-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/error-handling/render-until-error-expected.txt: Removed.
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/svg/path/property/priority-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-percentage-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-percentage-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/error-handling/render-until-error-expected.txt: Removed.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/svg/path/property/priority-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-lbse-text/imported/w3c/web-platform-tests/svg/path/error-handling/render-until-error-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk2-pixel/imported/w3c/web-platform-tests/svg/path/error-handling/render-until-error-expected.png: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/path/distance/pathlength-path-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/path/distance/pathLength-positive-percentage-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-expected.txt: Removed.
* LayoutTests/platform/mac/imported/w3c/web-platform-tests/svg/path/distance/pathLength-zero-percentage-expected.txt: Removed.

Canonical link: https://commits.webkit.org/289589@main
